sysprep/sysprep.ps1

```diff
@@ -243,6 +244,13 @@ $PSHome\powershell.exe -NoProfile -NoLogo -ExecutionPolicy Unrestricted -File "$
     }
   }
 
+  Write-Log 'Clearing MTLS MDS certs.'
```

Comment:
Trying to remember, will we also need to stop the agent from rotating in a new cert, or will this only happen on reboot/every N hours?

Even if it's every N hours I wonder if we'll ever run into a scenario where it chooses to rotate right when we go to capture it.

Reply:
Agent would remove old one and add newly fetched cert every time it starts up and then onwards rotate every 48hrs. 
I will reach out to you offline to make sure we cover all the cases.

Reply:
Stopping the agent is likely the safest approach since the system will shutdown at the end of the script, unless the noshutdown argument is used. Given that the agent keeps an internal schedule of operations there is now a chance of one of them triggering.

Stopping the agent right before `Write-Log 'Setting startup commands.'` would be my suggestion.

Reply:
Done